tap_quickbooks/schemas/bills.json

```diff
@@ -195,6 +195,46 @@
                   "object"
                 ]
               },
+              "CustomerRef": {
+                "properties": {
+                  "name": {
+                    "type": [
+                      "null",
+                      "string"
+                    ]
+                  },
+                  "value": {
+                    "type": [
+                      "null",
+                      "string"
+                    ]
+                  }
+                },
+                "type": [
+                  "null",
+                  "object"
+                ]
+              },
+              "ClassRef": {
```

Comment:
@dbshah1212 Create a ref object for the common dicts

Reply:
Done